libraries/rush-sdk/config/heft.json

```diff
@@ -6,6 +6,12 @@
 
   "extends": "@rushstack/heft-node-rig/profiles/default/config/heft.json",
   "eventActions": [
+    {
+      "actionKind": "deleteGlobs",
+      "heftEvent": "clean",
+      "actionId": "defaultClean",
+      "globsToDelete": ["dist", "lib", "lib-shim", "temp"]
```

Comment:
```suggestion
      "actionId": "deleteLibShim",
      "globsToDelete": ["lib-shim"]
```

Reply:
@iclanton doesn't `actionId` cause the inherited folders to get replaced rather than appended?

Reply:
It does seem to append -- so is there no way for an inherited action to remove an item from the array?
@iclanton